Makefile

```diff
@@ -3,9 +3,14 @@
 # Credits to Paul Sokolovsky (@pfalcon) for esp-open-sdk
 # Credits to Fabien Poussin (@fpoussin) for xtensa-lx106-elf build script
 #
+# (@Juppit) adaptation for gcc-4.9.2 and Cygwin
+# (@Juppit) touch makedir after make 
 
 
-VENDOR_SDK_VERSION = 1.4.0
+# for utils you need the tools pyinstaller and mcs
+BUILD_UTILS = n
```

Comment:
I would prefer to have this enabled by default. you get mcs and pyinstaller when configuring environment via scripts from env/